2-mandatory/4-people-i-know.js

```diff
@@ -379,13 +379,25 @@ First, I want you to find all of my friends who are 35 or older.
 
 let thirtyFiveOrOlder = [];
 
+for (let friend of friends) {
+  if (friend.age >= 35) {
+    thirtyFiveOrOlder.push(friend);
+  }
+}
+
 /*
```

Comment:
Well done ,  you can also use filter and save yourself some keystrokes, like below;

const thirtyFiveOrOlder = friend.filter(friend=> friend.age>=35);